tests/unit/registry.test.ts

```
@@ -2964,6 +2977,10 @@ const EXPECTED_EFFECTIVE_BUDGETS: Readonly<Record<string, number>> = {
   'exarchos_orchestrate.verify_delegation_saga': 2000,
   'exarchos_orchestrate.post_delegation_check': 2000,
   'exarchos_orchestrate.reconcile_state': 2000,
+  'exarchos_orchestrate.reconcile_worktrees': 2000,
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)